gen/gen.go

```diff
@@ -378,13 +399,15 @@ func (g *Generator) Write(w io.Writer) error {
 
 type typedef struct {
 	Name   string
+	Doc    string
 	Fields []field
 	Schema string
 }
 
-func newType(name string, fields []field, schema string) typedef {
+func newType(name string, doc string, fields []field, schema string) typedef {
```

Comment:
The linter is failing on this.
```suggestion
func newType(name, doc string, fields []field, schema string) typedef {
```